_posts/2015-02-03-4.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "Testing Simulated Kinect"
+date:   2015-02-03 10:00:04
+answerrostag: "simulation_kinect"
+track: [simulation]
+---
+## What is Kinect?
+{% include image_onrightoftext.html img="/assets/sim_kinect_camera.png" width="225" %}
+
+Kinect consists of three parts that work together: an RGB camera, a depth sensor, and a multi-array microphone. The TurtleBot uses Kinect to see the world in 3D and for detecting and tracking objects. 
+
+## Prerequisites
+
+First of all, make sure you have the openni packages installed:
+{% highlight sh %}
+sudo apt-get install ros-indigo-openni-*
+{% endhighlight %}
+
+You can check the active 3D sensor of Turtlebot by running:
+{% highlight sh %}
+echo $TURTLEBOT_3D_SENSOR
+{% endhighlight %}
```

Comment:
It would be great to give examples of a good output vs an incorrect output, both empty or set otherwise.